packages/presentation/lib/utils/responsive.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:flutter/material.dart';
+import 'package:presentation/utils/text_styles.dart';
+import 'dimens.dart';
+
+class Responsive {
+  const Responsive._();
+
+  static int getCrossAxisCount(BuildContext context) {
+    final double screenWidth = MediaQuery.of(context).size.width;
+    if (screenWidth > 1200) {
+      return 6;
+    } else if (screenWidth > 1000) {
+      return 5;
+    } else if (screenWidth > 800) {
+      return 4;
+    } else if (screenWidth > 600) {
+      return 3;
+    } else {
+      return 2;
+    }
+  }
+
+  static int getExpandedFlex(BuildContext context) {
+    final double screenWidth = MediaQuery.of(context).size.width;
+    if (screenWidth > 800) {
```

Comment:
since u almost everywhere check if width > 800, there is a sense to move it into a const, like minExpandedWidth or smth like this